pallets/parachain-staking/src/set.rs

```diff
@@ -76,10 +105,37 @@ impl<T: Ord> OrderedSet<T> {
 		self.0.binary_search(&value).is_ok()
 	}
 
+	/// Return if the set contains `value`
```

Comment:
Return... what? 🤣

Reply:
someone™ might have just copied the doc string from another function. Happens further below again a couple of times.

Reply:
We should probably just copy the docs from the rust doc


>Binary searches this ~~ordered VecDeque~~OrderedSet for a given element.
>
>If the value is found then Result::Ok is returned, containing the index of the matching element. ~~If there are multiple matches, then any one of the matches could be returned.~~ If the value is not found then Result::Err is returned, containing the index where a matching element could be inserted while maintaining sorted order.

Reply:
Fixed in af20eb0